<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="ui_grid_main">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="5" column="4" colspan="3">
     <layout class="QHBoxLayout" name="ui_grid_bt_action">
      <item>
       <widget class="QPushButton" name="ui_pb_branco">
        <property name="text">
         <string>Branco</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ui_pb_corrige">
        <property name="text">
         <string>Corrige</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ui_pb_confirma">
        <property name="text">
         <string>Confirma</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="Line" name="line_7">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="ui_lb_role">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="4" colspan="3">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="ui_lb_vote">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="Line" name="line_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="Line" name="line_8">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="Line" name="line_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="4" colspan="3">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="4" rowspan="2" colspan="3">
     <layout class="QGridLayout" name="ui_grid_bt_number">
      <item row="3" column="1">
       <widget class="QPushButton" name="ui_pb_0">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="ui_pb_7">
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="ui_pb_9">
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="ui_pb_8">
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="ui_pb_2">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="ui_pb_3">
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="ui_pb_5">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="ui_pb_1">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="ui_pb_4">
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="ui_pb_6">
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" rowspan="3" colspan="3">
     <layout class="QGridLayout" name="ui_grid_info">
      <item row="1" column="0">
       <widget class="QLabel" name="ui_lb_partido">
        <property name="text">
         <string>Partido:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="ui_lb_name">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="ui_lb_nome">
        <property name="text">
         <string>Nome:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="ui_lb_party">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2" colspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>753</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuA_es">
    <property name="title">
     <string>Ações</string>
    </property>
    <addaction name="ui_menu_start"/>
    <addaction name="ui_menu_stop"/>
    <addaction name="separator"/>
    <addaction name="ui_menu_new"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuSimula_o">
    <property name="title">
     <string>Simulação</string>
    </property>
    <addaction name="ui_menu_sim_gerar"/>
    <addaction name="ui_menu_sim_aplicar"/>
   </widget>
   <addaction name="menuA_es"/>
   <addaction name="menuSimula_o"/>
  </widget>
  <action name="ui_menu_start">
   <property name="text">
    <string>Iniciar Votação</string>
   </property>
  </action>
  <action name="ui_menu_stop">
   <property name="text">
    <string>Finalizar Votação</string>
   </property>
  </action>
  <action name="ui_menu_new">
   <property name="text">
    <string>Iniciar novo candidato</string>
   </property>
  </action>
  <action name="ui_menu_generate">
   <property name="text">
    <string>Gerar votos</string>
   </property>
  </action>
  <action name="ui_menu_sim_gerar">
   <property name="text">
    <string>Gerar Votos</string>
   </property>
  </action>
  <action name="ui_menu_sim_aplicar">
   <property name="text">
    <string>Aplicar Votos</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
